packages/api/app/Utils/statuses.js

```diff
@@ -178,6 +178,12 @@ const technologyLocationsTypes = {
 	WHERE_CAN_BE_APPLIED: 'where_can_be_applied',
 };
 
+const tokenTypes = {
+	CONFIRM_AC: 'confirm-ac',
+	NEW_EMAIL: 'new-email',
+	RESET_PW: 'reset-pw',
```

Comment:
```suggestion
	CONFIRM_ACCOUNT: 'confirm-account',
	REGISTER_ACCOUNT: 'register-account',
	RESET_PASSWORD: 'reset-password',
```

Reply:
This names are used in all API, stored in database too.

Reply:
Exactly. It needs to be a more meaningful name. Otherwise, years from now it will be difficult to know what it is

Reply:
Ok, for `REGISTER_ACCOUNT` no make sense, because `NEW_EMAIL` token type is for change email, no register a new account. I suggest `CHANGE_EMAIL`: `change-email`

Reply:
Okay. It makes sense :+1: